solidity/random-beacon/test/Utils.test.ts

```diff
@@ -0,0 +1,75 @@
+import { ethers } from "hardhat"
+import { expect } from "chai"
+import { hashDKGMembers } from "./utils/dkg"
+
+describe("utils/dkg", () => {
+  context("hashDKGMembers", () => {
+    const members = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9]
+
+    context("when there are no misbehaved members", () => {
```

Comment:
We're missing `it` blocks. We could rename the contexts to e.g. 
`it("calculates hash when there are no misbehaved members", () => {`

Reply:
2ca346e